bundles/org.openhab.binding.tasmotaplug/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -10,6 +10,7 @@
 		<description>
 			Tasmota Smart Plug
 		</description>
+		<semantic-equipment-tag>PowerOutlet</semantic-equipment-tag>
```

Comment:
Please see ongoing discussion here: https://github.com/openhab/openhab-addons/pull/18549#discussion_r2042678410

Reply:
+1 vote for `SmartPlug`